content/chainguard/chainguard-images/working-with-images/image-stigs.md

```diff
@@ -0,0 +1,128 @@
+
+DISA understands that containers have different requirements than traditional operating systems. In an effort to highlight these differences, the DOD DevSecOps Initiative released the [Container Hardening Process Guide](https://dl.dod.cyber.mil/wp-content/uploads/devsecops/pdf/Final_DevSecOps_Enterprise_Container_Hardening_Guide_1.2.pdf) which describes the Initiative's approach to hardening images and how other agencies should handle applying STIGs to containers.
+
+[Appendix C](https://dl.dod.cyber.mil/wp-content/uploads/devsecops/pdf/Final_DevSecOps_Enterprise_Container_Hardening_Guide_1.2.pdf#%5B%7B%22num%22%3A65%2C%22gen%22%3A0%7D%2C%7B%22name%22%3A%22XYZ%22%7D%2C70%2C720%2C0%5D) of the Hardening Process Guide highlights an important point about STIG compliance for containers:
+
+_"With a properly locked down hosting environment, containers inherit most of the security
+controls and benefits from infrastructure to host OS-level remediation requirements."_
+
+Deploying containers on a STIG hardened host provides many of the security features that are difficult or sometimes impossible to implement inside a container. What's left then is the application-level security configuration — in particular vulnerability remediation — which Chainguard provides through our guaranteed vulnerability remediation SLAs. 
+
+Once a container has been deployed, you can use tools such as [OpenSCAP](http://www.open-scap.org/tools/) to validate that the container was appropriately hardened. Those scans will return [false positives](/chainguard/chainguard-images/recommended-practices/false-results/) any time they detect missing components inside the container which are inherited from the host where the container is running. As stated in [Appendix D](https://dl.dod.cyber.mil/wp-content/uploads/devsecops/pdf/Final_DevSecOps_Enterprise_Container_Hardening_Guide_1.2.pdf#%5B%7B%22num%22%3A67%2C%22gen%22%3A0%7D%2C%7B%22name%22%3A%22XYZ%22%7D%2C70%2C720%2C0%5D) of the DSO Initiative's Container Hardening Process Guide:
+
+_"If an OpenSCAP scan returns noncompliant result(s), always evaluate the validity of those findings. False positives are common within major host OS-based containers, as the security profiles normally account for all host-level controls potentially not applicable to a container build (e.g., GUI, CAC authentication, etc.). In addition to false positives, many of the base OS STIG requirements are not applicable in the containers either."_
+
+To sum up, it can be difficult to determine exactly which results are false positives and which ones should be applied to a container. 
```

Comment:
I'm not sure we need this callout. The problem with the existing GPOS SRG for wolfi wasn't false _positives_, but _not applicables_.

The STIG initiative we undertook was more around "teaching" `openscap` and other xccdf compliant scanners how to look for the right wolfi specific GPOS things.

Reply:
Fair enough!